packages/plugins/src/foundry/utils/index.ts

```diff
@@ -1168,3 +1168,26 @@ export const findFullyQualifiedName = (
   // If we make it to this point, we couldn't find a fully qualified name for this init code.
   return undefined
 }
+
+/**
+ * Returns the fully qualified name that corresponds to a given address. Returns `undefined` if no
+ * fully qualified name exists for the address. This function will only find a fully qualified name
+ * if it corresponds to a contract deployed in one of the actions.
+ */
+export const findFullyQualifiedNameForAddress = (
```

Comment:
This is the un-implemented function that's causing CI to fail. I figured there's no point implementing this function because the `RawActionInput` will have a different structure when I make the PR that transitions to the state diff. In other words, I'd need to completely redo this function in the next PR. (The biggest difference is that the `RawActionInput` probably wont' have an `additionalContracts` array in the state diff).